float64[] Xwaypoints
float64[] Ywaypoints
---
bool res
---
geometry_msgs/TransformStamped vicon_data
